src/transformState.js

```diff
@@ -6,6 +6,29 @@
  */
 function transformState(state, actions) {
   // write code here
-}
+  for (const act of actions) {
```

Comment:
Don't shorten the words.

```suggestion
  for (const action of actions) {
```